nle/scripts/play.py

```diff
@@ -67,7 +68,18 @@ def get_action(env, action_mode, is_raw_env):
     return action
 
 
-def play(env, mode, ngames, max_steps, seeds, savedir, no_render, render_mode, debug):
+def play(
+    env,
+    mode,
+    ngames,
+    max_steps,
+    seeds,
+    savedir,
+    no_render,
+    render_mode,
+    print_frames_separately,
+    **kwargs,
```

Comment:
What do we use the **kwargs for?

Reply:
the `debug` flag is created but does nothing in this function because it has been used previously. however this fn is called with `play(**vars(flags))`, so it needs a home here. the home i thought would be clearer in `**kwargs`, but i almost put `**_`